### PR TITLE
19

### LINKED ISSUE #19: checkstyle-suppressions.xml:32: move message text to...
The puzzle `16-6e4c5756` from #16 has to be resolved: 

https://github.com/Blamer-io/bot/blob/b37ea9254f8c7bd7f6bb35bc4668d82c4bd45243/checkstyle-suppressions.xml#L32-L32

The puzzle was created by @h1alexbel on 27-Apr-23. 

 role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by l3r8yJ**
closes #19

@h1alexbel take a look, please

---

## PR-Codex overview
This PR adds new functionality to the bot, including new messages and commands, and updates some dependencies and configurations.

### Detailed summary
- Added `StartMessageGenerator` and `TokenMessageGenerator` classes to handle `/start` and `/token` commands respectively
- Added `answers.properties` file to store message and command properties
- Updated `BotApplicationEntryTest` to use `@TestWithSpringContext` annotation
- Updated `application.yaml` and `codecov.yaml` with new environment variables
- Updated some dependencies and their versions in `pom.xml` file
- Updated `banner.txt` files with new ASCII art and version information
- Removed some checkstyle suppressions in `checkstyle-suppressions.xml` file

> The following files were skipped due to too many changes: `src/test/java/io/blamer/bot/answer/generator/impl/StartMessageGeneratorTest.java`, `src/test/java/io/blamer/bot/answer/generator/impl/TokenMessageGeneratorTest.java`, `src/test/java/io/blamer/bot/answer/generator/impl/RegistryMessageGeneratorTest.java`, `src/main/java/io/blamer/bot/answer/generator/impl/RegistryMessageGenerator.java`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/main/java/io/blamer/bot/answer/generator/impl/RegistryMessageGenerator.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  }
+
+  /**
+   * Detaches token form messages like this.
+   * <pre>/registry token</pre>
+   *
+   * @param update The update to get token
+   * @return Result as string
+   */
+  private static String detachToken(final Update update) {
```
Comment (h1alexbel) [inline src/main/java/io/blamer/bot/answer/generator/impl/RegistryMessageGenerator.java:80]: @l3r8yJ maybe we should stay away from static methods and private ones?
Open for discussion

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -1,3 +1,15 @@
 bot:
   token: ${BOT_TOKEN}
   name: ${BOT_NAME}
+  answers:
```
Comment (h1alexbel) [inline src/main/resources/application.yaml:4]: @l3r8yJ Maybe we should store message content in `properties`, I think application.yml is designed for config only

File: src/main/java/io/blamer/bot/answer/generator/impl/RegistryMessageGenerator.java (hunk 1 above)
Comment (l3r8yJ) [reply src/main/java/io/blamer/bot/answer/generator/impl/RegistryMessageGenerator.java:80]: @h1alexbel this method for a temporary solution
When the hub is created, we will rework this class a bit, wdyt

Comment (h1alexbel) [reply src/main/java/io/blamer/bot/answer/generator/impl/RegistryMessageGenerator.java:80]: @l3r8yJ got it. Can you create a ticket for it?

Comment (l3r8yJ) [reply src/main/java/io/blamer/bot/answer/generator/impl/RegistryMessageGenerator.java:80]: @h1alexbel sure

**Comment by l3r8yJ** [comment]
@h1alexbel take a look, please

**Comment by l3r8yJ** [comment]
@codex what changed in RegistryMessageGenerator?

**Comment by l3r8yJ** [comment]
/codex do you see some code issues in this pull request?

**Comment by l3r8yJ** [comment]
/codex do you see some code issues in this code diff?

**Review by h1alexbel — APPROVED** [review]
lgtm

**Comment by l3r8yJ** [comment]
@rultor merge

**Pull request merged**